docs/app/pages/showcase/visualizations/partition-map/partition-map.component.html

```diff
@@ -0,0 +1,71 @@
+<div class="chart-container">
+    <ux-partition-map class="partition-map" [dataset]="dataset" [colors]="colors">
+
+        <ng-template
+            #partitionMapSegment
+            let-segment="segment"
+            let-value="value"
+            let-expanded="expanded"
+            let-color="color"
+            let-children="children">
+
+            <div #customSegment
```

Comment:
I think it would be good if we made this wider, it looks a little bit cramped:

![image](https://user-images.githubusercontent.com/20795331/72337039-bf9fec00-36b9-11ea-9283-44ca128ded36.png)

It might also be good to give the title a bit more prominence with a darker colour, and reduce the spacing above the title a bit. There also seems to be more space at the bottom of the popover than the top, perhpas something like this? 


![image](https://user-images.githubusercontent.com/20795331/72337008-b44cc080-36b9-11ea-9ece-05a6bb1d60ca.png)